src/css/style.scss

```diff
@@ -248,3 +271,11 @@ h1 {
   font-weight: bold;
   font-family: sans-serif;
 }
+
+.hidden {
+  visibility: none;
+}
```

Comment:
Cannot remember what we are using this hidden visibility for? Did we end up using only the blur?

Reply:
@carflor I dont think we are using the hidden class for anything at this point.